webpack.config.js

```diff
@@ -59,10 +74,6 @@ module.exports = (_env, argv = {}) => {
           use: [
             {
               loader: MiniCssExtractPlugin.loader,
-              options: {
-                hmr: true,
-                esModule: true,
-              },
```

Comment:
Check the changelog for mini-css-extract-plugin [here](https://github.com/webpack-contrib/mini-css-extract-plugin/blob/master/CHANGELOG.md#breaking-change) for why the options were removed.